### DISCUSSION TITLE
Is there a way to change the logo ?

### DISCUSSION THREAD

**Discussion by MGNXYZ** [searxng/searxng#1413] [Q&A] [answered]
Hi !
I'm looking for a way to change the logo i already changed the favicon, i have tryed to change the logo in the exact same way as i did for the favicon but it's isn't working any idea ?

**Comment by tiekoetter** [comment, marked as answer]
You would have to replace the searxng.png (https://github.com/searxng/searxng/blob/master/searx/static/themes/simple/img/searxng.png).

Make sure that you either replace it after every update or directly in the source code (by running your own fork for example).

**Reply by MGNXYZ** [reply]
Thank's !

**Reply by raskitoma** [reply]
> You would have to replace the searxng.png (https://github.com/searxng/searxng/blob/master/searx/static/themes/simple/img/searxng.png).
> 
> Make sure that you either replace it after every update or directly in the source code (by running your own fork for example).

Hi, I'm using the docker version. I tried from mounting using volumes (my custom image to the path you mentioned), replacing the image as well with no luck.  

Which method are you using to deploy your instance?

**Reply by raskitoma** [reply]
Don't mind me... was a cache issue... thanks Cloudflare!

**Reply by return42** [reply]
Just a piece of advice: it's not recommended to use Cloudflare as reverse proxy because it MITM every request, which is a disaster for user's privacy.

**Reply by Tazzetiel** [reply]
> You would have to replace the searxng.png (https://github.com/searxng/searxng/blob/master/searx/static/themes/simple/img/searxng.png).
> 
> Make sure that you either replace it after every update or directly in the source code (by running your own fork for example).

How would I go about this if i had it hosted on a cloud server (linode)? I do plan to get myself a home-hosting server set up, but getting parts still is an issue...
I installed via [this guide video](https://www.youtube.com/watch?v=ifT6npY39Dw), as reference. Help would be absolutely appreciated (please keep in mind I am an absolute newbe if it comes to this kind of thing.

**Reply by return42** [reply]
> please keep in mind I am an absolute newbe if it comes to this kind of thing.

Please understand that we cannot provide any training here. We can also not make detailed statements about a specific hoster or (video) guides not made by us. Discussions here should be reserved for the product -- the SearXNG server. 

There is a large community that can help with many questions --> I would recommend to discuss wider aspects on our chat: [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org).  

Thanks for your understanding :+1:
